Tests/SourceKitDTests/SourceKitDTests.swift

```diff
@@ -13,25 +13,23 @@
 import LSPTestSupport
 import SourceKitD
 import SKCore
+import SKSupport
 import TSCBasic
 import ISDBTibs
 import ISDBTestSupport
+import Foundation
 import XCTest
 
-import enum TSCUtility.Platform
+import class TSCBasic.Process
```

Comment:
How come we need to import `TSCBasic.Process` where we didn’t need to import it before?

Reply:
Because `TSCUtility` dragged it along for us.  Something something `@_implementationOnly` all the things and explicit imports.